index.js

```diff
@@ -1396,6 +1396,11 @@ function Runtime() {
         const protocols = properties.protocols || [];
         const methods = properties.methods || {};
         const events = properties.events || {};
+        const canonSelNames = new Set(
```

Comment:
```suggestion
        const selectorNames = new Set(
```

Think we want to spell this out, as this variable has a larger scope it deserves a more verbose name. I guess we can drop the `canonical` part since these are selector names, and the keys in `methods` are method names, so we don't need to disambiguate them.

Reply:
Agreed!

Reply:
Or perhaps: `supportedSelectors` or `selectors`. Specifying that it's containing names of selectors seems a bit redundant since selectors are usually referred to by their human-readable string representation.

(Should also update the reference below btw, I was a bit sloppy here and only added one suggestion 😅)

Reply:
No worries, should've caught it 😅